opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -63,11 +64,11 @@ impl UserEventsExporter {
         _provider_group: ProviderGroup,
         exporter_config: ExporterConfig,
     ) -> Self {
-        let mut options = eventheader_dynamic::Provider::new_options();
-        options = *options.group_name(provider_name);
+        let options = eventheader_dynamic::Provider::new_options();
         let mut eventheader_provider: eventheader_dynamic::Provider =
             eventheader_dynamic::Provider::new(provider_name, &options);
         Self::register_keywords(&mut eventheader_provider, &exporter_config);
+        otel_debug!(name: "UserEventsExporter.Created", config = format!("{:?}", exporter_config), provider_name = provider_name, provider_group = format!("{:?}", options));
```

Comment:
could we print the exact tracepoint names in an internal log? That'll make it easy to setup perf/other tools to listen to one, instead of users trying to figure out the exact name?

Reply:
ignore, I think this is done already in another place.